### PR TITLE
Specify HC retention period

### PR DESCRIPTION
**Pull request by cicku**

### CODE REVIEW

File: data/plans.json [hunk 1]
```diff
@@ -2603,9 +2603,9 @@
         "analytics": {
           "title": "Analytics",
```
Comment (victor-perov) [inline data/plans.json]: If the idea to show how far back in time each plan can request data, then title is misleading. I would use something like "Maximum time range" or something

**Review by victor-perov — CHANGES_REQUESTED** [review]
Could we please rename that field and put appropriate title to avoid confusion?

File: data/plans.json (hunk 1 above)
Comment (cicku) [reply data/plans.json]: @angelampcosta Need your opinion on how we manage the comparison here. If you are ok with the title then I will do it.

Comment (angelampcosta) [reply data/plans.json]: Sounds good to me. We can go with Viktor's suggestion.

**Review by angelampcosta — APPROVED** [review]

**Comment by angelampcosta** [comment]
Hi @victor-perov The PR was updated. Could you please review? Thanks!

**Comment by victor-perov** [comment]
@angelampcosta @cicku 

Thanks for pinging me on this ticket.

What should we do in this case with lines 2626-2633 of the same config? Where we do use "analytics" as availability for LoadBalancing analytics.

In my understanding you're trying to update/alter meaning of the object attributed without updating all use-cases of it.

**Review by victor-perov — CHANGES_REQUESTED** [review]
I think we should either get a linked PR / issue to get the same applied (see my comment above) for LB Analytics or change it as part of this PR too.

**Pull request closed without merging**